pep-3156.txt

```diff
@@ -1647,15 +1720,31 @@ callback-based frameworks like Twisted.  After converting a coroutine
 into a Task, callbacks can be added to the Task.
 
 To convert a coroutine into a task, call the coroutine function and
-pass the resulting coroutine object to the ``asyncio.Task()``
-constructor.  You may also use ``asyncio.async()`` for this purpose.
+pass the resulting coroutine object to the ``loop.create_task()``
+method.  You may also use ``asyncio.ensure_future()`` for this purpose.
 
 You may ask, why not automatically convert all coroutines to Tasks?
 The ``@asyncio.coroutine`` decorator could do this.  However, this would
 slow things down considerably in the case where one coroutine calls
 another (and so on), as switching to a "bare" coroutine has much less
 overhead than switching to a Task.
 
+A Task is inherited from ``Future`` but adds new methods:
```

Comment:
The `Future` interface is fairly broad; I think the `Task` interface that we want to document and expose here is narrower than `Future`'s. For example, a `Task` is always associated with some runner that will call `set_result` or `set_exception` on it; calling those methods from a `Task` retrieved from `current_task()` will cause problems. We should specify which methods may be called here (and therefore must be implemented by alternative Task implementations). Just `cancel()`? the `result()` accessors? adding callbacks?